.circleci/real_config.yml

```diff
@@ -1513,14 +1513,15 @@ jobs:
 
   test-e2e-react:
     docker:
-      - image: cimg/node:20.9.0
+      - image: mcr.microsoft.com/playwright:v1.40.0-focal
```

Comment:
It's probably quicker to run with the Circle convenience images. Does the Microsoft-hosted image have an advantage?

Reply:
Yes, for the playwright, `cimg/node:20.9.0` is not straightforward to setup. We used to MS image, but at some point, it was replaced with `cimg/node:20.9.0` with Playwright version downgrade. That's not what we want because that prevents us to use the latest playwright version.